### ISSUE TITLE
fog gem required but not installed

### ISSUE THREAD

**Issue by bcbroom** [dblock/dzt#9]
Its quite possible I have a borked system, but every time I tried to run dzt, I got errors, such as 

```
Brians-Mac-mini:~ brian$ dzt
/Users/brian/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- fog (LoadError)
	from /Users/brian/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/brian/.rvm/gems/ruby-2.1.1/gems/dzt-0.2.1/bin/dzt:4:in `<top (required)>'
	from /Users/brian/.rvm/gems/ruby-2.1.1/bin/dzt:23:in `load'
	from /Users/brian/.rvm/gems/ruby-2.1.1/bin/dzt:23:in `<main>'
	from /Users/brian/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/brian/.rvm/gems/ruby-2.1.1/bin/ruby_executable_hooks:15:in `<main>'
Brians-Mac-mini:~ brian$ subl /Users/brian/.rvm/gems/ruby-2.1.1/gems/dzt-0.2.1/bin/dzt
```

After trying a bunch of stuff, and going in circles, I tried `gem install fog`, and then the dzt command worked. Is it possible its not setup correctly in the gem dependencies? Is this a rvm issue?

Anyway, it seems to be working now, but thought I would mention it in case it helped anyone else.

**Comment by dblock** [comment]
Fixed in https://github.com/dblock/dzt/commit/bb0a311cb34d4c548ee9f4f36dd16fc8113efd83.

**Comment by dblock** [comment]
Released in 0.2.2.
